sys/errors.h

```diff
@@ -1,4 +1,3 @@
-#pragma once
```

Comment:
You remove this line. This must always be included at the top of header files so that there aren't multiple redefinitions